database/seeders/ProveedorSeeder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\Proveedor;
+
+class ProveedorSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Proveedor::factory(1)->create();
```

Comment:
Hey Herbert necesitamos crear 5 registros de prueba, esito despues. Looks Good To Me